ocs_ci/deployment/azure.py

```diff
@@ -0,0 +1,13 @@
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class AZUREIPI:
```

Comment:
You are failing without this definition mainly cause we are trying to instantiate deployer class even for testing.

I am fixing this issue here:  https://github.com/red-hat-storage/ocs-ci/pull/2307/files#diff-484462fced51d1a06b1d93b4a44dd535R1048  . But if we will do deployment here It's OK to have this class already defined and the work you done in the map...